hadoop-hdds/managed-rocksdb/src/main/java/org/apache/hadoop/hdds/utils/db/managed/ManagedBloomFilter.java

```diff
@@ -20,14 +20,20 @@
 
 import org.rocksdb.BloomFilter;
 
+import static org.apache.hadoop.hdds.utils.db.managed.ManagedRocksObjectUtils.assertClosed;
+import static org.apache.hadoop.hdds.utils.db.managed.ManagedRocksObjectUtils.formatStackTrace;
+import static org.apache.hadoop.hdds.utils.db.managed.ManagedRocksObjectUtils.getStackTrace;
+
 /**
  * Managed BloomFilter.
  */
 public class ManagedBloomFilter extends BloomFilter {
 
+  private final StackTraceElement[] elements = getStackTrace();
```

Comment:
I guess it's more obvious if stacktrace is an `Optional<StackTraceElement[]>` to it's not always available (disabled in prod).

Reply:
Good point, but let me annotate as `@Nullable` instead.